MAUI/DataGrid/paging.md

```diff
@@ -0,0 +1,344 @@
+    {
+        InitializeComponent();
+        dataPager.NumericButtonsGenerateMode = DataPagerNumericButtonsGenerateMode.Auto;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> The size of the `SfDataPager` is automatically adjusted based on the available screen size if the view cannot accommodate the numeric buttons specified in the `NumericButtonCount` property.
+
+## Customizing button size and font size of pager buttons
+
+The `SfDataPager` button is loaded with a default width and height of 40. The default button font size of `SfDataPager` is 14. You can customize the button size and font size by setting the desired values for the `SfDataPager.ButtonSize` and `SfDataPager.ButtonFontSize` properties, respectively.
+
+{% tabs %}
+{% highlight xaml %}
+<pager:SfDataPager x:Name="dataPager"
+                   PageSize="15"
+                   ButtonSize="60"
+                   ButtonFontSize="21"
+                   Source="{Binding OrdersInfo}"/>
+</pager:SfDataPager>
+{% endhighlight %}
+{% highlight c# %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        dataPager.ButtonSize = 60;
+        dataPager.ButtonFontSize = 21;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Button size and Font size of .NET MAUI DataGrid](Images\paging\maui-datagrid-buttonsize-and-fontsize.png)
+
+## DataPagerStyle
+
+The data grid allows you to change its appearance by creating a style class that overrides the `DataPagerStyle` and then assigning it to the `SfDataPager.DefaultStyle` property.
+  
+To apply custom style, follow the code example:
+
+{% tabs %}
+{% highlight xaml %}
+<pager:SfDataPager x:Name="dataPager"
+                         PageSize="15"
+                         Source="{Binding OrdersInfo}">
+                <pager:SfDataPager.DefaultStyle >
+                    <pager:DataPagerStyle NumericButtonSelectionBackgroundColor="Red"
+                                          NumericButtonBackgroundColor="GreenYellow"
+                                          NavigationButtonBackgroundColor="Black"
+                                          NavigationButtonIconColor="White">
+                    </pager:DataPagerStyle>
+                </pager:SfDataPager.DefaultStyle>
+</pager:SfDataPager>
+{% endhighlight %}
+{% highlight c# %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        dataPager.DefaultStyle.NumericButtonSelectionBackgroundColor = Colors.Red;
+        dataPager.DefaultStyle.NumericButtonBackgroundColor = Colors.GreenYellow;
+        dataPager.DefaultStyle.NavigationButtonBackgroundColor = Colors.Black;
+        dataPager.DefaultStyle.NavigationButtonIconColor = Colors.White;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+The following picture shows the customize styles of data pager:
+
+![DataPager style .NET MAUI DataGrid](Images\paging\maui-datagrid-pager-style.png)
+
+### Limitations

```

Comment:
Need to add topics that cover all the design spec properties in the paging UG.

- Events
- Display mode
- Ellipsis mode
- Move to pages programmatically
- Display template to the navigation buttons
- Orientation

Reply:
Added the mentioned topics.